ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -398,21 +398,27 @@ Rectangle {
         }
 
         if (event.matches(StandardKey.Paste)) {
-            messageInputField.remove(messageInputField.selectionStart, messageInputField.selectionEnd)
+            const clipboardImage = QClipboardProxy.imageBase64
+            if (clipboardImage !== "") {
```

Comment:
You could do a check for `QClipboardProxy.hasImage` here instead of fetching the image only to discover it's empty :)

Reply:
Good call!